### PR TITLE
Adding warnings in panorama-commit-status

### PR DESCRIPTION
**Pull request by GalRabin**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/23421
fixes:  https://github.com/demisto/etc/issues/23759
## Description
Adding warnings in `panorama-commit-status`,
This enhancement will allow easier debugging on Demisto in case of unsuccessful EDL Object commit, as a playbook can be marked as completed while there is an issue with the commit/push.

*** @yaakovi - Currently secrets detection is broken, so not able to merge, we need to understand how to merge it.

## Screenshots
1. EC example:
image

2. Added also in war-room output:
image

## Minimum version of Demisto
- [ ] 4.5.0
- [x] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by GalRabin** [comment]
@kirbles19 
I will appreciate your review on the following changes:
1. [README](https://github.com/demisto/content/blob/pan_os_warning/Packs/Legacy/Integrations/Panorama/Panorama.yml#L2199)
2. [YML](https://github.com/demisto/content/blob/pan_os_warning/Packs/Legacy/Integrations/Panorama/README.md#28-get-the-commit-status-for-a-configuration)

File: Packs/Legacy/Integrations/Panorama/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
 ## [Unreleased]
--
-
+- Adding warnings to panorama-commit-status command.
```
Comment (yaakovi) [inline Packs/Legacy/Integrations/Panorama/CHANGELOG.md:2]:
```suggestion
Added warnings to the ***panorama-commit-status*** command.
```

File: Packs/Legacy/Integrations/Panorama/README.md [hunk 2]
```diff
@@ -2341,6 +2341,11 @@
 <td style="width: 81px;">string</td>
```
Comment (yaakovi) [inline Packs/Legacy/Integrations/Panorama/README.md:2341]: regenerate the docs in markdown format

File: Packs/Legacy/Integrations/Panorama/Panorama.yml [hunk 3]
```diff
@@ -1756,13 +1756,13 @@ script:
       required: false
       secret: false
     - default: false
+      defaultValue: any
       description: A comma-separated list of application object namesfor the rule
```
Comment (kirbles19) [inline Packs/Legacy/Integrations/Panorama/Panorama.yml:1760]: A comma-separated list of application object names for the rule.

**Comment by EliorKedar** [comment]
@gal001 @yaakovi looks good.

**Comment by kirbles19** [comment]
@GalRabin Looks good.

**Comment by GalRabin** [comment]
Opened a separate issue for generating documentation - https://github.com/demisto/etc/issues/23796

**Pull request merged**